locale/en/knowledge/child-processes/how-to-spawn-a-child-process.md

```diff
@@ -14,7 +14,7 @@ The simplest way is the "fire, forget, and buffer" method using `child_process.e
 
 *Note that the examples you will see in this article are all Linux-based. On Windows, you need to switch these commands with their Windows alternatives.*
 
-Let us take a look at an example:
+Let's take a look at an example:
```

Comment:
Let us is fine, but both are equivalent AFAICT so I'll leave this to someone else

Reply:
```suggestion
Take a look at an example:
```